.travis.yml

```diff
@@ -2,34 +2,24 @@ dist: bionic
 language: python
 python: 3.7
 
-matrix:
+jobs:
```

Comment:
Does this have any significance or are they the same?

Reply:
As far as I understand it, they are different indeed: In a matrix build, every possible combination of configurations is run, jobs are more straight-forward. Since in our case each jobs is quite different from the others, a matrix build does not make much sense.

Reply:
Got it, jobs does sound more appropriate.